packages/amplication-server/src/core/action/action.service.ts

```diff
@@ -196,6 +196,6 @@ export class ActionService {
     message: string,
     meta: JsonValue = {}
   ): Promise<void> {
-    await this.log(step, EnumActionLogLevel.Info, message, meta);
+    await this.log(step, EnumActionLogLevel.Error, message, meta);
```

Comment:
@manulpatel thanks for the contribution but this is the correct change unfortunately.
This change will cause all logs to be logged as errors. 
Moreover, the method name clearly state the intention `logInfo` and it should raise some questions about if that is or not the correct place to change the code.

Reply:
Very sorry @overbit for the mistake. I found the function responsible to log a PR failure is [here](https://github.com/amplication/amplication/blob/b10f2d21ff6deef833a159563dbd06c7c5922e45/packages/amplication-server/src/core/build/build.service.ts#L642C2-L642C2). Now to log at error level do we need to change input parameters to include error ?

Reply:
@manulpatel yep, this comment might help https://github.com/amplication/amplication/issues/6118#issuecomment-1596769566